WreckEngine/TriPlay.cpp

```diff
@@ -28,7 +28,7 @@ TriPlay::TriPlay(GLprogram prog) : Game(6)
     addState(menuState);
     menuState->addEntity(make_shared<LogicEntity>(menu_update));
     
-    auto menuText = make_shared<TextEntity>("Press space to begin.", "QuartzMS.ttf", Text::Justify::MIDDLE, Text::Justify::MIDDLE, 48);
+    auto menuText = make_shared<TextEntity>("Press space\nto begin.", "QuartzMS.ttf", Text::Justify::MIDDLE, Text::Justify::MIDDLE, 48);
```

Comment:
I know this is here for testing purposes, but make sure to revert it once we're about ready to merge, it looks weird